transform/operations.go

```diff
@@ -318,6 +318,30 @@ func (s *stringToInteger) transform(raw interface{}) (interface{}, error) {
 	return strconv.Atoi(str)
 }
 
+// stringToFloat is a transformOperation which takes a string and converts it into a float
+type stringToFloat struct {
+}
+
+func (s *stringToFloat) init(args map[string]string) error {
+	return nil
+}
+
+func (s *stringToFloat) transform(raw interface{}) (interface{}, error) {
+	_, ok := raw.(float64)
```

Comment:
is this first bit necessary?

Reply:
It is. The tests keep failing. Took a while before I realized that raw was already pass in as float.